tests/framework-tests/core/data/messages1_en.json

```diff
@@ -0,0 +1,3 @@
+{
+    "courses": "These courses will require a lot of grading"
+}
```

Comment:
Can you add a new line char at the end of these new json files as well as tests/framework-tests/core/js/FluidRequestsTests.js